route-rs-runtime/src/link/composite/drop_link.rs

```diff
@@ -119,6 +119,11 @@ mod tests {
 
     #[test]
     fn finishes_with_wait() {
+        run_finishes_with_wait();
+    }
+
+    #[tokio::main]
```

Comment:
Can we annotate `finishes_with_wait` with multiple macros so we don't need to dispatch to an extra function?

Reply:
This is changed in the next commit where I rework the testing harness.  I just reworked run_link and such. See the test harness commit. I think that resolves this.

Reply:
Oh yes I believe you're right. I was leaving comments as I read through each commit (they're a little more bite-sized, focused), so we can resolve them if you already addressed the comment.